runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixOverlay.java

```diff
@@ -55,8 +55,19 @@ private TearsOfGuthixOverlay(TearsOfGuthixPlugin plugin)
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		Color BLUE_TEARS_ALPHA = config.getBlueTearsColor();
+		Color GREEN_TEARS_ALPHA = config.getGreenTearsColor();
+		Color BLUE_TEARS_COLOR = new Color(BLUE_TEARS_ALPHA.getRed(), BLUE_TEARS_ALPHA.CYAN.getGreen(), BLUE_TEARS_ALPHA.CYAN.getBlue());
+		Color GREEN_TEARS_COLOR = new Color(GREEN_TEARS_ALPHA.getRed(), GREEN_TEARS_ALPHA.getGreen(), GREEN_TEARS_ALPHA.getBlue());

```

Comment:
You should use camelCase notation here as those aren't static variables anymore. I'd propose calling the first two `[green/blue]TearsFill` and the other two `[green/blue]TearsBorder` as they are used for the fill and border of the pie timers.

For the border colors, you could also use `ColorUtil.colorWithAlpha(color, 0xFF)`.

Also I just noticed L60 looks broken, it has some Cyan poking through, `BLUE_TEARS_ALPHA.CYAN.getXX()` .

Reply:
Yeah, noticed the extra CYAN. I was about to replace the line with `ColorUtil.colorWithAlpha(color, 255)`. Is the preference on using hexadecimal here?

Reply:
I saw it used with `0xFF` in TextComponent.java [here](https://github.com/runelite/runelite/blob/c9cd33a8dd4eff05159fac664420006a74baec99/runelite-client/src/main/java/net/runelite/client/ui/overlay/components/TextComponent.java#L104), but I think either is fine.

Reply:
If you're going to use `, 100)` in the config fields you should stay consistent and use `, 255)` here I suppose.

Reply:
Oops, didn't see this reply until after I pushed up `0xFF`. I can go either way, replacing `100` with `0x64` or `0xFF` to `255`.

Reply:
I'd say go for the `255`.

At the same time, I noticed you named the green border color `greenTearsColor`, while the blue one is correctly named `blueTearsBorder`.

I think after that it's going to look pretty good to merge, but Adam will likely be the final judge.

Reply:
Ah, yes. Apologies for the sloppiness here, my attention is a bit split right now. I appreciate the thorough review and guidance, thank you!